crosscompile.bash

```diff
@@ -8,7 +8,7 @@ VERSION="`git describe --abbrev=0 --tags --exact-match || git rev-parse --short
 BUILD_DATE="`date -u +%Y%m%d%.%H%M%S`"
 export VERSION_STRING="$VERSION ($BUILD_DATE)"
 LOGGLY_TOKEN="469973d5-6eaf-445a-be71-cf27141316a1"
-LDFLAGS="-w -X main.version $VERSION -X main.buildDate $BUILD_DATE -X main.logglyToken LOGGLY_TOKEN"
+LDFLAGS="-w -X main.version $VERSION -X main.buildDate $BUILD_DATE -X github.com/getlantern/flashlight/logging.logglyToken $LOGGLY_TOKEN"
```

Comment:
Hmm... that's a thing. Since that's the only thing one needs to post to loggly I was going to suggest we remove it from the public view and add it to too-many-secrets but then I realized it will be embedded into the binary anyway, so I guess we may as well leave it public for now.